ONSdigital/eq-author-app
# Amend content for scroll pane

Updates the content of the component doc

## How to review
- Load up storybook `yarn storybook`
- Check the scroll pane component is under components
- The content reflects that of the google doc in the JIRA ticket
